### ISSUE TITLE
NoSuchMethodError in IntelliJ Idea 12

### ISSUE THREAD

**Issue by TachikomaGT** [hsz/idea-gitignore#71]
After updating to plugin version 0.8, when .gitignore file is open, I got many exceptions with following stack trace:

```
java.lang.NoSuchMethodError: com.intellij.lang.parser.GeneratedParserUtilBase.current_position_(Lcom/intellij/lang/PsiBuilder;)I
	at mobi.hsz.idea.gitignore.parser.GitignoreParser.gitignoreFile(GitignoreParser.java:162)
	at mobi.hsz.idea.gitignore.parser.GitignoreParser.parse_root_(GitignoreParser.java:43)
	at mobi.hsz.idea.gitignore.parser.GitignoreParser.parse(GitignoreParser.java:36)
	at com.intellij.psi.tree.ILazyParseableElementType.doParseContents(ILazyParseableElementType.java:64)
	at com.intellij.psi.tree.IFileElementType.parseContents(IFileElementType.java:43)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.e(LazyParseableElement.java:165)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:209)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:34)
	at com.intellij.psi.impl.source.tree.SharedImplUtil.getFirstChild(SharedImplUtil.java:44)
	at com.intellij.psi.impl.source.PsiFileImpl.getFirstChild(PsiFileImpl.java:746)
	at com.intellij.psi.SingleRootFileViewProvider.findElementAt(SingleRootFileViewProvider.java:440)
	at com.intellij.psi.SingleRootFileViewProvider.findElementAt(SingleRootFileViewProvider.java:397)
	at com.intellij.codeInsight.daemon.impl.CollectHighlightsUtil.a(CollectHighlightsUtil.java:158)
	at com.intellij.codeInsight.daemon.impl.CollectHighlightsUtil.findCommonParent(CollectHighlightsUtil.java:139)
	at com.intellij.codeInsight.daemon.impl.CollectHighlightsUtil.getElementsInRange(CollectHighlightsUtil.java:53)
	at com.intellij.codeInsight.daemon.impl.CollectHighlightsUtil.getElementsInRange(CollectHighlightsUtil.java:45)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.collectInformationWithProgress(LineMarkersPass.java:113)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:58)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:62)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass$1$1.run(PassExecutorService.java:357)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1182)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass$1.run(PassExecutorService.java:348)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:226)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:345)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:321)
	at com.intellij.concurrency.JobLauncherImpl$3.call(JobLauncherImpl.java:145)
	at com.intellij.concurrency.JobLauncherImpl$3.call(JobLauncherImpl.java:142)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at com.intellij.concurrency.PrioritizedFutureTask.access$101(PrioritizedFutureTask.java:31)
	at com.intellij.concurrency.PrioritizedFutureTask$1.run(PrioritizedFutureTask.java:70)
	at com.intellij.concurrency.PrioritizedFutureTask.run(PrioritizedFutureTask.java:113)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
```

IntelliJ Idea: 12.1.7
JRE: 1.6.0_65
OS: Mac OS X 10.9

**Comment by Kwestor** [comment]
I have same problem with Idea 12.1.7, could you please look into this?

**Comment by hsz** [comment]
Fixed with release `v0.8.1`
